Formula/servefiles.rb

```diff
@@ -0,0 +1,35 @@
+class Servefiles < Formula
+  desc "Simple Python script for serving local files to FBI's remote installer."
```

Comment:
what is FBI in this context?

Reply:
A program, installed on the 3DS (so it doesn't make sense to package FBI in homebrew), to install homebrew software because Nintendo didn't include one themselves.

Reply:
How about something like
```ruby
desc "Serve local files to the Nintendo 3DS via the FBI remote installer"